app/components/onboarding_instructions/onboarding_instructions.js

```diff
@@ -0,0 +1,36 @@
+import { Controller } from 'stimulus';
```

Comment:
I would love to see tests for the ever increasing JavaScript code base. I don't know how to properly test StimulusJS. In VueJS, testing components is straight forwards with https://vue-test-utils.vuejs.org/.